song-server/src/test/java/bio/overture/song/server/service/AnalysisServiceTest.java

```diff
@@ -729,9 +730,11 @@ public void testVariantCallAnalysisMissingSamplesException() {
   @Transactional
   public void testAnalysisIdDneException(){
     val nonExistentAnalysisId = analysisGenerator.generateNonExistingAnalysisId();
-    assertSongError(() -> service.checkAnalysisAndStudyRelated(DEFAULT_STUDY_ID, nonExistentAnalysisId),
+    SongErrorAssertions
+        .assertSongErrorRunnable(() -> service.checkAnalysisAndStudyRelated(DEFAULT_STUDY_ID, nonExistentAnalysisId),
         ANALYSIS_ID_NOT_FOUND);
-    assertSongError(() -> service.checkAnalysisExists(nonExistentAnalysisId), ANALYSIS_ID_NOT_FOUND);
+    SongErrorAssertions
```

Comment:
seems like this static method is used in alot of places and not statically imported. go through all the files and statically import it